scripts/apps/authoring/attachments/AttachmentsEditorDirective.ts

```diff
@@ -2,5 +2,13 @@
 export default function AttachmentsEditorDirective() {
     return {
         templateUrl: 'scripts/apps/authoring/attachments/attachments.html',
+        link: function(scope, elem, attrs, controllers) {
+            if (attrs.readOnly != null) {
+                scope.$watch(() => scope.$eval(attrs.readOnly), (value) => {
```

Comment:
just wondering what's the difference to do `scope.$watch(attrs.readOnly, ...)`

Reply:
I don't know, I'm not too savvy in angular details. I though one can only do `scope.$watch(prop)` when `prop` is a part of the scope. In my case, it's coming as an attribute which I needed to evaluate in order to get a value and after I've done that, I simply used the same in the watch. I'll see if the shortcut you're proposing works.